guides/source/configuring.md

```diff
@@ -676,7 +676,10 @@ Enables the use of SHA256 fingerprints in asset names. Set to `true` by default.
 
 #### `config.assets.debug`
 
-Disables the concatenation and compression of assets. Set to `true` by default in `development.rb`.
+Disables the concatenation and compression of assets. Set to `false` by default.
+
+Before Rails 7.0, new applications had `config.assets.debug = true` in the automatically generated
+`config/environments/development.rb` file. After Rails 7.0, the configuration line has been removed.
```

Comment:
We should probably use a table for this instead, similar to framework defaults:
```suggestion
| Starting with version | The value in development.rb |
| --------------------- | -------------------- |
| (original)            | `true`               |
| 7.0                   |  Removed           |
```

Reply:
This would be more confusing in my opinion, as we would be visually conflating `load_defaults` with this.

Since we have versioned documentation already, I don't think there's a need to specify in which version a template changed (as opposed to `load_defaults` of old versions which are still accessible on newer versions of Rails, so it makes sense to document them all).

Reply:
I think we could at least mention the guide version of Rails to make it clear these might change and can differ if the reader has a different version:
```
Disables the concatenation and compression of assets. Set to `true` by default in `development.rb` for Rails 8.0.
```

Reply:
I agree that using a table could be a little bit confusing at first sight. I however like the idea of at least mentioning the Rails version where the configuration was present in the `development.rb`.

What about this:
```
Disables the concatenation and compression of assets. Defaults to `false`, but before 
Rails 7.0 it was set to `true` in the automatically generated `config/environments/development.rb` file.